### DISCUSSION TITLE
远程工作、开源协同与饱和沟通 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#71] [Comments]
# 远程工作、开源协同与饱和沟通 | 夜天之书

上周末在给 Apache Ratis 的代码库上 Maven Wrapper 的时候，项目作者 @szetszwo 对 Pull Request (PR) 当中代码的合规问题和实现问题提出了一些问题。虽然这些问题并不难回答，但是我第一时间的反应却是“这么明显的答案，实在是懒得回复”。进而我又想到，如果是线下集中办公或者面对面 workshop 的情形，这些问题不到五分钟就能讨论清楚，而这两类不同环

https://www.tisonkun.org/2023/03/20/remote-over-communication/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
理念先进,
其实就是路径依赖演示...
制度制定者如果一开始对领域问题比较清晰,
并有全套模式应对,
那么无论公布什么制度, 几次事件后, 大家发现可以很好的完成反应,
就对制度本身有了信心,
然后, 才愿意开始在制度基础上进行创新;
而不是相反, 开始积极 over 制度本身
